### PR TITLE
Client-side parameter value update for Kafka TS

### PR DESCRIPTION
**Pull request by harshini-rangaswamy**
# What changed, and why it matters

Added a section to include info about client-side parameter value. 

Ref: [DOC-704](https://aiven.atlassian.net/browse/DOC-704)

[DOC-704]: https://aiven.atlassian.net/browse/DOC-704?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/products/kafka/howto/enable-kafka-tiered-storage.rst [hunk 1]
```diff
@@ -55,6 +57,12 @@ Configuring default retention policies at service-level
 
 Additionally, you can configure the retention policies from the :ref:`Tiered storage overview <modify-retention-polices>` page.
 
+Configure client-side parameter
```
Comment (MichaelTansiniAiven) [inline docs/products/kafka/howto/enable-kafka-tiered-storage.rst:60]: can it be clear that this is optional though recommended?

File: docs/products/kafka/howto/kafka-tiered-storage-get-started.rst [hunk 2]
```diff
@@ -2,13 +2,14 @@
 Get started with tiered storage for Aiven for Apache Kafka® 
 ====================================================================
 
+.. important:: 
+    
+    Aiven for Apache Kafka® tiered storage is a :doc:`limited availability feature </docs/platform/concepts/beta_services>`. If you're interested in trying out this feature, contact the sales team at sales@Aiven.io.
```
Comment (MichaelTansiniAiven) [inline docs/products/kafka/howto/kafka-tiered-storage-get-started.rst:7]: early

File: docs/products/kafka/howto/enable-kafka-tiered-storage.rst (hunk 1 above)
Comment (harshini-rangaswamy) [reply docs/products/kafka/howto/enable-kafka-tiered-storage.rst]: Update the text to reflect this. Please review.

File: docs/products/kafka/howto/kafka-tiered-storage-get-started.rst (hunk 2 above)
Comment (harshini-rangaswamy) [reply docs/products/kafka/howto/kafka-tiered-storage-get-started.rst:7]: fixed.

**Review by ArthurFlageul — APPROVED** [review]

**Review by jeqo — APPROVED** [review]

**Pull request merged**